packages/bee-q/src/components/avatar/scss/bq-avatar.scss

```
@@ -5,23 +5,34 @@
 @import './bq-avatar.variables';
 
 :host {
-  @apply inline-block;
+  @apply relative inline-block;
+}
+
+.avatar {
+  @apply relative overflow-hidden;
+  @apply bg-[var(--bq-avatar-background)];
+
+  border: var(--bq-avatar--border-width) var(--bq-avatar--border-style) var(--bq-avatar--border-color);
```

Try to use Tailwind CSS notations as much as possible

```suggestion
  @apply relative overflow-hidden bg-[var(--bq-avatar-background)];
  @apply border-[color:var(--bq-avatar--border-color)] border-[length:var(--bq-avatar--border-width)];

  border-style: var(--bq-avatar--border-style);
```